<launch>
<arg name="wheel_base" default="1">
<arg name="track_width" default="0.5">

<arg name="l_rope_drive_topic" default="">
<arg name="r_rope_drive_topic" default="">
<arg name="l_top_wheel_topic" default="">
<arg name="r_top_wheel_topic" default="">



<node pkg="cmd_vel_controller" 
    type="cmd_vel_controller.py" 
    name="cmd_vel_controller" 
    output="screen" 
    args="$(arg wheel_base) $(arg track_width)"/>

</launch>